### ISSUE TITLE
Guidelines for annotatiing own dataset to finetune coreferee pretrained model

### ISSUE THREAD

**Issue by Tanmay98** [richardpaulhudson/coreferee#16]
Hi, 
I am interested in annotating my own custom dataset for finetuning existing pretrained model. 
I have tried reviewing some of the public datasets available like
 - ParCor
 - LitBank
 - Gap Corefernce
 - OntoNotes / Conll-2012

I am little confused as all are not similar to each other. Can you suggest me some basic guidelines for annotation. It would be great help. 
Thanks in advance!

**Comment by Tanmay98** [comment]
So I tried to figure out the annotation done in LitBank dataset, I am able to produce the brat.ann and brat.txt files for NER but how do you combine them using cl-coref-annotator to produce the .tsv files.

Can you help or give some suggestions on how to proceed?

**Comment by Tanmay98** [comment]
I have annotated mu own custom dataset for coreferee, but Its a small dataset for finetuning. The code provided helps me to finetune over the current public coreferee model right?
Thanks in advance!

**Comment by richardpaulhudson** [comment]
Unfortunately Coreferee does not support finetuning an existing model with new training data, although you can certainly train a new model with the data that was used to train an existing model plus your new traning data, which should have the same effect.

If your main aim is to get Coreferee working with a custom spaCy model, please follow the instructions at https://github.com/explosion/coreferee/issues/13. 

If, on the other hand, you wish to add training data to improve the accuracy of the coreference resolution itself, the annotated data is loaded in using the loader classes in https://github.com/explosion/coreferee/blob/master/coreferee/training/loaders.py. Looking at these loader classes, you will notice that they do not require all features of each of the supported annotation formats: if you are annotating data specifically for Coreferee, you only need to provide those features that the loader class in question examines. You may also find it quicker to produce training data in some other format or in a custom format and to write your own loader class to load it in. As you can see in the command under point 9 of https://github.com/explosion/coreferee/#adding-support-for-a-new-language, it is possible to combine training data in different formats, so whatever format you use you will have no problems combining it with the existing (standard) training data.

**Comment by Tanmay98** [comment]
@richardpaulhudson Thanks for your response. I see, unfortunately the custom NER spacy model I trained was on small dataset as I finetuned it over the existing "en_wb_sm" model so basically I dont have any existing big dataset.

So let's suppose I want to train the coreferee model for my custom entities, how much training data would be enough for getting some decent baseline custom coreferee model? or is not feasible to re train a new coreferee model for new NER entities?

Thanks in advance!
